packages/paste-core/components/input/__tests__/input.test.tsx

```diff
@@ -60,7 +60,7 @@ describe('Input render', () => {
 
   it('should render with suffix', (): void => {
     const {asFragment} = render(
-      <Theme.Provider theme="console">
+      <Theme.Provider theme="default">
```

Comment:
If any of these snapshots are already covered by 

a) a chromatic VRT test and 
b) a test asserting any specific DOM attributes we expect (assert Input is aria-described by helpText for example)

Lets just remove them as they are redundant at that point.

Reply:
Any that remain truly uncovered, lets swap them to using the customization provider and the mock global theme we created to prevent test flaps when we modify design tokens

Reply:
Removed bc they were redundant!